tmpl.html

```diff
@@ -102,7 +114,36 @@ <h3>command</h3>
 
 $plotly_score_steps
 
+/*var data = [{*/
+  /*type: 'table',*/
+  /*header: {*/
+    /*values: $header,*/
+    /*align: ["left", "center"],*/
+    /*line: {width: 1, color: '#506784'},*/
+    /*fill: {color: '#119DFF'},*/
+    /*font: {family: "Arial", size: 12, color: "white"}*/
+  /*},*/
+  /*cells: {*/
+    /*values: $culist,*/
+    /*align: ["left", "center"],*/
+    /*line: {width: 1, color: "#506784"},*/
+    /*fill: {color: ['#25FEFD', 'white']},*/
+    /*font: {family: "Arial", size: 11, color: ["#506784"]}*/
+  /*}*/
+/*}]*/
+
+/*Plotly.newPlot('table-div', data);*/
+
+$$(document).ready(function() {
+    $$('#example').DataTable( {
```

Comment:
use `jQuery` instead of `$$` and rename `#example` to `#goi_table` or something informative.